### PR TITLE
bugfix: oracle and postgreSQL can't query column info(#3496)

### LINKED ISSUE #3496: global_table table or TRANSACTION_NAME column not found
- [x] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
在使用 Postgres 作为 db 时，启动时提示："global_table table or TRANSACTION_NAME column not found"

### Ⅱ. Describe what happened
启动日志
```
2021/1/27 下午6:12:37SLF4J: A number (17) of logging calls during the initialization phase have been intercepted and are
2021/1/27 下午6:12:37SLF4J: now being replayed. These are subject to the filtering rules of the underlying logging system.
2021/1/27 下午6:12:37SLF4J: See also http://www.slf4j.org/codes.html#replay
2021/1/27 下午6:12:3710:12:37.251  INFO --- [                     main] io.seata.server.Server                   : The server is running in container.
2021/1/27 下午6:12:3710:12:37.263  INFO --- [                     main] io.seata.config.FileConfiguration        : The file name of the operation is registry
2021/1/27 下午6:12:3710:12:37.265  INFO --- [                     main] io.seata.config.FileConfiguration        : The configuration file used is /seata-server/resources/registry.conf
2021/1/27 下午6:12:3710:12:37.350  INFO --- [                     main] io.seata.config.FileConfiguration        : The file name of the operation is file.conf
2021/1/27 下午6:12:3710:12:37.351  INFO --- [                     main] io.seata.config.FileConfiguration        : The configuration file used is /seata-server/resources/file.conf
2021/1/27 下午6:12:3710:12:37.766  INFO --- [                     main] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2021/1/27 下午6:12:3710:12:37.779  WARN --- [                     main] i.s.s.s.db.store.LogStoreDataBaseDAO     : global_table table or TRANSACTION_NAME column not found
2021/1/27 下午6:12:3710:12:37.927  INFO --- [                     main] i.s.core.rpc.netty.NettyServerBootstrap  : Server started, listen port: 8091
2021/1/27 下午6:12:3710:12:37.950  WARN --- [                     main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2021/1/27 下午6:12:3710:12:37.950  INFO --- [                     main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2021/1/27 下午6:12:3710:12:37.985  INFO --- [                     main] c.netflix.config.DynamicPropertyFactory  : DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@b86de0d
2021/1/27 下午6:12:3710:12:37.989  WARN --- [                     main] c.n.d.internal.util.Archaius1Utils       : Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
2021/1/27 下午6:12:3810:12:38.000  INFO --- [                     main] .p.EurekaConfigBasedInstanceInfoProvider : Setting initial instance status as: STARTING
2021/1/27 下午6:12:3810:12:38.008  WARN --- [                     main] c.n.d.internal.util.Archaius1Utils       : Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
2021/1/27 下午6:12:3810:12:38.038  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Initializing Eureka in region us-east-1
2021/1/27 下午6:12:3810:12:38.394  INFO --- [                     main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2021/1/27 下午6:12:3810:12:38.394  INFO --- [                     main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2021/1/27 下午6:12:3810:12:38.531  INFO --- [                     main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2021/1/27 下午6:12:3810:12:38.531  INFO --- [                     main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2021/1/27 下午6:12:3810:12:38.785  INFO --- [                     main] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Application is null : false
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Application version is -1: true
2021/1/27 下午6:12:3810:12:38.803  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2021/1/27 下午6:12:38log4j:WARN No appenders could be found for logger (com.netflix.discovery.shared.MonitoredConnectionManager).
2021/1/27 下午6:12:38log4j:WARN Please initialize the log4j system properly.
2021/1/27 下午6:12:38log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
2021/1/27 下午6:12:3910:12:39.030  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : The response status is 200
2021/1/27 下午6:12:3910:12:39.034  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Starting heartbeat executor: renew interval is: 30
2021/1/27 下午6:12:3910:12:39.046  INFO --- [                     main] c.n.discovery.InstanceInfoReplicator     : InstanceInfoReplicator onDemand update allowed rate per min is 4
2021/1/27 下午6:12:3910:12:39.057  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1611742359051 with initial instances count: 13
2021/1/27 下午6:12:3910:12:39.057  INFO --- [                     main] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1611742359057, current=UP, previous=STARTING]
2021/1/27 下午6:12:3910:12:39.058  INFO --- [-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_xxxxx:8091: registering service...
2021/1/27 下午6:12:3910:12:39.089  INFO --- [-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_xxxxx:8091 - registration status: 204
```

### Ⅲ. Describe what you expected to happen
消除数据库相关异常

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅴ. Anything else we need to know?
no

### Ⅵ. Environment:
使用的镜像：`seataio/seata-server:latest` (DIGEST：`seataio/seata-server@sha256:45a933a6df705f244b65713012d66ed4360587adc43a387810dd25db4102ecb9`)
数据库：PostgreSQL 10.14 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39), 64-bit
数据库初始化脚本（语句中的 `public` 替换为自定义 schema，在自定义 Schema 下操作）：https://github.com/seata/seata/blob/develop/script/server/db/postgresql.sql

相关配置
```
registry.type=eureka

store.mode = db
store.db.datasource = druid
store.db.dbType = postgresql
store.db.driverClassName = org.postgresql.Driver
store.db.url = jdbc:postgresql://xxxxx/myschema
store.db.user = user
store.db.password = pwd
```

### PR DESCRIPTION
**Pull request by dmego**
### Ⅰ. Describe what this PR did
When using Oracle or PostgreSQL as the database,  can't query column info

### Ⅱ. Does this pull request fix one issue?

fixes #3496

### CODE REVIEW

File: server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java [hunk 1]
```diff
@@ -492,8 +492,27 @@ private ColumnInfo queryTableStructure(final String tableName, String colName) {
...
+                rs = ps.executeQuery();
+                String schema = null;
+                if (rs.next()) {
+                    schema = rs.getString(1);
+                }
+                return schema;
+            } catch (SQLException e) {
+                throw new StoreException(e);
+            } finally {
+                IOUtil.close(rs, ps, conn);
```
Comment (xingfudeshi) [inline server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:511]: conn should be closed here?

Comment (xingfudeshi) [reply server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:511]: image

Comment (dmego) [reply server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java]: solved

**Review by slievrly — APPROVED** [review]
LGTM, thanks for u contributions.

**Review by xingfudeshi — APPROVED** [review]
LGTM.

**Pull request merged**